spec/features/shared_examples/cherrypicking.rb

```diff
@@ -260,6 +273,14 @@
               generated_file = DownloadHelpers.downloaded_file("#{batch_id}_batch_#{destination_barcode}_#{pick_number_index}.gwl")
               generated_lines = generated_file.lines
 
+              # check count of controls present in destination file lines is correct
+              # NB. Tecan file has additional plate barcode lines at bottom of file for beds so add 1
```

Comment:
Maybe we could define this +1 in a variable so the name describes why is added (like tecan_aditional_lines=1 and then +tecan_aditiona_lines in line 281)

Reply:
Ok will make this clearer.
Tecan has extra lines at the bottom of the file, one per plate, e.g. SCRC1 = DN1112124A
This is for the Tecan software on the robot to assign the beds I think.
Hamilton/Beckman files don't have this.